os2forms_payment.services.yml

```diff
@@ -3,6 +3,8 @@ services:
     arguments:
       - '@request_stack'
       - '@http_client'
+      - '@entity_type.manager'
+      - '@logger.factory'
```

Comment:
You can autowire services in Drupal (cf. https://www.drupal.org/node/3218156). This makes it much easier to change signatures of service constructors:

``` yaml
  Drupal\os2forms_payment\Helper\PaymentHelper:
    autowire: true
```

Run `drush cache:rebuild` when changes are made to arguments.